### PR TITLE
Deploy kube-state-metrics-app as unique app on CP

### PR DESCRIPTION
**Pull request by wtty-fool**
Towards https://github.com/giantswarm/giantswarm/issues/13332

### CODE REVIEW

File: integration/config/.env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+TEST_GIT_COMMIT=$(git rev-parse HEAD)
+TEST_GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+TEST_APP_VERSION=$(cat helm/kube-state-metrics-app/Chart.yaml | grep appVersion | cut -d':' -f2 | tr -d '[:blank:]"')
```
Comment (mig4) [inline integration/config/.env:3]:
```suggestion
TEST_APP_VERSION=$(grep appVersion helm/kube-state-metrics-app/Chart.yaml | cut -d':' -f2 | tr -d '[:blank:]"')
```

Comment (mig4) [reply integration/config/.env:3]: Or even:

```suggestion
TEST_APP_VERSION=$(awk -F: '$1 == "appVersion" { print $2 }' helm/kube-state-metrics-app/Chart.yaml | tr -d '[:blank:]"')
```

File: integration/config/.env [hunk 2]
```diff
@@ -0,0 +1,5 @@
+TEST_GIT_COMMIT=$(git rev-parse HEAD)
+TEST_GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
```
Comment (mig4) [inline integration/config/.env:2]: Does this work in CI? Asking as branch info doesn't seem to be accessible sometimes.

Comment (wtty-fool) [reply integration/config/.env:2]: Tested, works :+1:

**Review by mig4 — APPROVED** [review]
Lovely ❤️

**Pull request merged**